src/sql-parser/tests/sqlparser_common.rs

```diff
@@ -3718,13 +3718,6 @@ fn parse_create_source_csv_custom_delim() {
     }
 }
 
-#[test]
-fn parse_missing_format() {
```

Comment:
I think that this test should have moved to some semantic analysis phase (maybe an SLT) instead of just being removed? Do we already have this tested somewhere?

Reply:
Ah, good point. I meant to do that and forgot about it :)